### PR TITLE
Adds donate page

### PR DESCRIPTION
**Pull request by dqgorelick**
Needs following links to be updated: 

- final t-shirt fundraiser link
- final youtube video link

Then is ready to merge with approval from @sruti

TODO: 

- [x] update with final copy changes 
- [x] Add “Donate” to menu below “About Us ->” 
- [x] Add link to “Support” button on About page 
- [x] Add “Donate” to bottom bar under “Get Involved”

### CODE REVIEW

File: app/assets/scripts/views/donate.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                        If you contribute through a DAF, please{' '}
+                        <a href="info@openaq.org">send us an email</a> so we can
+                        so we can update the campaign total!
+                      </span>
+                    </p>
+                  </div>
+                </div>
+              </header>
+              <figure className="fold__media donate-info-wrapper">
+                <YoutubeEmbed embedId="C66UmEWoPos" />
```
Comment (dqgorelick) [inline app/assets/scripts/views/donate.js:59]: @sruti YouTube video ID goes here when we have the final video ready

File: app/assets/scripts/views/donate.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                    to fight air inequality by releasing some new OpenAQ swag
+                    available for a limited time only! Your purchase will help
+                    support our work in democratizing access to air quality data
+                    to enable community action to fight air pollution. Please
+                    join us in wearing OpenAQ shirts with pride, and spreading
+                    the word of Clean Air for All!
+                  </p>
+                  <p className="fold__action">
+                    <a
+                      href="https://www.customink.com/fundraising/2021-openaq-summer-swag-fundraiser?side=front&type=1&zoom=false"
```
Comment (dqgorelick) [inline app/assets/scripts/views/donate.js:164]: @sruti final t-shirt link goes here when it is ready

File: app/assets/scripts/views/donate.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+            <div className="inner">
+              <header className="fold__header donate-window">
+                <div>
+                  <h1 className="fold__title">Donate now to OpenAQ!</h1>
+                  <div className="prose prose--responsive">
+                    <p>
+                      Help support our mission to end air pollution through air
+                      quality data and community. Donate below or directly with{' '}
+                      <a
+                        href="https://www.dafdirect.org/DAFDirect/daflink?_dafdirect_settings=NDc1MzI0MTcyXzIxMTFfZDA0ODQ3MzYtOGJkOS00YTJjLWFmOTctM2Q2MGE3YTg4MDU4&designatedText=Z2VuZXJhbCBvcmdhbml6YXRpb25hbCBzdXBwb3J0ICh1bnJlc3RyaWN0ZWQp&amountValue="
```
Comment (dqgorelick) [inline app/assets/scripts/views/donate.js:34]: @sruti please confirm the correct link we will want for DAF

File: app/assets/scripts/views/donate.js (hunk 2 above)
Comment (sruti) [reply app/assets/scripts/views/donate.js:164]: This is final!

File: app/assets/scripts/views/donate.js (hunk 3 above)
Comment (sruti) [reply app/assets/scripts/views/donate.js:34]: This is correct

**Review by sruti — CHANGES_REQUESTED** [review]
Looks like there are some link errors being thrown preventing the build: https://github.com/openaq/openaq.org/actions/runs/904632748

**Comment by sruti** [comment]
Actually linting was fine, the tests failed. Need to update the tests: https://github.com/openaq/openaq.org/actions/runs/906400959

**Pull request merged**